sdks/python/tests/library_integration/openai/test_openai.py

```diff
@@ -306,6 +316,8 @@ async def async_f():
                         end_time=ANY_BUT_NONE,
                         project_name=ANY_BUT_NONE,
                         spans=[],
+                        model="gpt-3.5-turbo-0125",
```

Comment:
As soon as openai updates the version - the test will fail. We can:
1. relax is to ANY_BUT_NONE, and then check separately that it starts from `gpt-3.5-turbo-`
2. implement a new comparison helper in `testlib.any_compare_helpers`that will work approximately like that:
```python
"gpt-3.5-turbo-0125" == AnyString(starting_with="gpt-3.5-turbo-")
```
I'd say that the second approach would be more useful since it aligns with our testing paradigm better and we'll be able to reuse it (if we need, we can extend it with `ending_with` or `containing` args in the future).